ofmeet/src/java/org/jivesoftware/openfire/plugin/ofmeet/ConfigServlet.java

```diff
@@ -202,6 +232,107 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         }
     }
 
+	private void handleAuthenticatedUser(JSONObject config, HttpServletRequest request, String xmppDomain, JSONObject conferences)
+	{
+		String userName = request.getUserPrincipal().getName();
+
+		config.put( "id", userName + "@" + xmppDomain );
+
+		try {
+			User user = XMPPServer.getInstance().getUserManager().getUser(userName);
+
+			config.put( "emailAddress", user.getEmail() );
+			config.put( "nickName", user.getName() );
```

Comment:
Agreed. I have an immediate requirement to label my conference participant with their OF user directory names.